src/classes/ChildCreatorController.cls

```diff
@@ -4,17 +4,8 @@ public with sharing class ChildCreatorController {
 	public static SObject[] saveItems(SObject[] itemToBeSavedList, SObject[] itemToBeRemovedList){
 		system.debug('@@itemToBeSavedList '	+itemToBeSavedList);
 		system.debug('@@itemToBeRemovedList '	+itemToBeRemovedList);
-		List<SObject> toInsertList = new List<SObject>();
-		List<SObject> toUpdateList = new List<SObject>();
-		for(SObject item: itemToBeSavedList){
-			if(item.Id == null){
-				toInsertList.add(item);
-			} else {
-				toUpdateList.add(item);
-			}
-		}
-		insert toInsertList;
-		update toUpdateList;
+		
+		upsert itemToBeSavedList;
```

Comment:
@jongpie That's the error:

![errorlightning](https://cloud.githubusercontent.com/assets/12348998/19920522/7eb88c52-a0d9-11e6-9a3e-2beebcde0c40.PNG)

Welcome to Lightning!